applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -72,20 +72,23 @@ struct temp_cap_storage {
 	struct bt_audio_codec_cap codec[CONFIG_CODEC_CAP_COUNT_MAX];
 };
 
-static struct le_audio_unicast_server unicast_servers[CONFIG_BT_MAX_CONN];
+/* Since there is no subgroups for CIG we will use 1 as a hard coded value */
+static struct le_audio_unicast_server unicast_servers[CONFIG_BT_ISO_MAX_CIG][1][CONFIG_BT_MAX_CONN];
 
-K_MSGQ_DEFINE(cap_start_msgq, sizeof(uint8_t), CONFIG_BT_MAX_CONN, sizeof(uint8_t));
+K_MSGQ_DEFINE(cap_start_msgq, sizeof(struct stream_index), CONFIG_BT_ISO_MAX_CHAN, 4);
```

Comment:
Magic value

Reply:
This is the alignment value, and almost all uses of K_MSGQ_DEFINE has the value 4. We could of course use `sizeof(uint32_t)` instead